apps/ui/src/views/Space/Settings.vue

```diff
@@ -270,9 +270,8 @@ async function handleSettingsSave() {
 }
 
 function handleControllerSave(value: string) {
-  if (!isController.value) return;
+  if (!isOwner.value) return;
```

Comment:
That seems to work now, but we have to be careful with `isOwner` because it's lazy async computed property (it was made lazy because it triggered lots of requests checking for ownership on unrelated screens).

If it wasn't used in the UI it could return unexpected value.